text/0013-schema-enums.md

```diff
@@ -0,0 +1,97 @@
+# Enums in Schemas
+
+## Related issues and PRs
+
+- Reference Issues: 
+- Implementation PR(s): 
+
+## Timeline
+
+- Start Date: 2023-06-30
+- Date Entered FCP:
+- Date Accepted:
+- Date Landed:
+
+## Summary
+
+Add support for enumerations of `bool`, `string`, and `long` types in schemas, and use this information when doing policy validation and schema-based-parsing.
+
+## Basic example
+
+Maintaining backward compatibility with JSON schema as much as possible, we can match its enum feature. For example:
```

Comment:
"backward compatibility" might be a strange way to word this, perhaps just "compatibility" or "inspiration from"?